lib/workers/repository/update/branch/index.ts

```diff
@@ -540,6 +550,7 @@ export async function processBranch(
         branchExists: true,
         prNo: branchPr?.number,
         result: BranchResult.Pending,
+        updateBranchFingerprint: !!commitSha,
```

Comment:
```suggestion
        commitSha,
```

Let's just return the commitSha, instead of making this update branch function suddenly be aware of branch fingerprint logic